### PR TITLE
Fixed leaf pom.xml files

### PR DESCRIPTION
**Pull request by danielylee01**
We fixed the pom.xml dependencies in the leaf files, removed the parents, and merged changes from master. To note: this process also removed bookshelf/pom.xml and bookshelf-standard/pom.xml, which Averi edited earlier.

### CODE REVIEW

**Comment by lesv** [comment]
@danielylee01 / @SaketramDurbha Could you make things compile?

File: appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml [hunk 1]
```diff
@@ -25,11 +25,11 @@ limitations under the License.
...
-  <parent>                              <!-- Get parameters and allow bulk testing -->
-    <artifactId>getting-started-java</artifactId>
-    <groupId>com.example</groupId>
-    <version>1.0.0</version>
-    <relativePath>../..</relativePath>
+  <!-- Parent POM defines common plugins and properties. -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.17</version>
```
Comment (lesv) [inline appengine-standard-java8/helloworld-gae-javasdk-tools/pom.xml:32]: 1.0.18

**Comment by kurtisvg** [comment]
@lesv - The current problem with the tests is that they are missing the selenium webdriver. This wasn't showing up before because the root parent pom had tests disabled, so things were compiling but not executing the tests. 

@danielylee01 and @SaketramDurbha are going to get the webdriver added to the testing docker, and then file issues for any tests that are broken outside of pom modifications.

**Comment by danielylee01** [comment]
@googlebot I consent.

**Comment by lesv** [comment]
I'd merge this, but you still have it in DRAFT.  Please let me know if it's ok to merge.

**Comment by lesv** [comment]
Actually, I'm going to merge.

**Pull request merged**